tests/functional_tests/ffront_tests/test_unrollPowOp.py

```diff
@@ -0,0 +1,139 @@
+import pathlib
+
+from eve.pattern_matching import ObjectPattern
+from functional.ffront import common_types as ct, field_operator_ast as foast
+from functional.ffront.func_to_foast import UnrollPowerOp
+
+
+def _make_power_testee(pow_n: str):
+    _set_source_location = foast.SourceLocation(
+        line=106, column=12, source=str(pathlib.Path(__file__).resolve())
+    )
+
+    testee = foast.BinOp(
+        right=foast.Constant(
+            dtype=ct.ScalarType(kind=ct.ScalarKind.INT),
+            location=_set_source_location,
+            value=pow_n,
+            type=ct.ScalarType(kind=ct.ScalarKind.INT),
+        ),
+        left=foast.Name(
+            id="a",
+            type=ct.DeferredSymbolType(constraint=None),
+            location=_set_source_location,
+        ),
+        op=foast.BinaryOperator.POW,
+        location=_set_source_location,
+        type=ct.DeferredSymbolType(constraint=None),
+    )
+    return testee
+
+
+def test_power_1():
```

Comment:
Please use https://docs.pytest.org/en/6.2.x/parametrize.html to avoid duplicating the tests that barely change the value of the exponent.